lineage/figures/figure7.py

```diff
@@ -10,19 +10,37 @@
 from .Fig_Gen import Lineage_Length, Lineages_per_Population_Figure, AIC
 from ..tHMM_utils import getAccuracy, getAIC
 from ..Lineage_utils import remove_singleton_lineages
-
+import matplotlib
+import matplotlib.ticker
+matplotlib.use('Agg')
+import matplotlib.ticker as ticker
+from .Matplot_gen import moving_average
 
 def makeFigure():
```

Comment:
```suggestion
def makeFigure7():
```

Reply:
Actually, don't think this is recommended. This might break something with the way Dr. Meyer has it set up. Ignore for now.